FLEDGE.md

```diff
@@ -297,9 +302,11 @@ The `ads` list contains the various ads that the interest group might show.  Eac
 
  * `metadata`: Arbitrary metadata that can be used at bidding time.
 
+ * `creativeScanningMetadata`: A USVString, no character limit. If `sendCreativeScanningMetadata` is set to true in the auction config, then this string -- along with other metadata as described in [2.1 Initiating an On-Device Auction](#21-initiating-an-on-device-auction) -- is sent as part of the scoring signals fetch request. This provides a way for buyers to convey additional information that sellers need in order to scan an advertisement, such as the associated advertiser domain, which isn't captured in the `renderURL` and other fields sent on the scoring signals fetch request.
```

Comment:
would you mind line wrapping this (and any other lines this PR adds or modifies)?
can we linkify USVString like we do above on line 287?
true->`true`
Can we elaborate slightly on "other metadata"?  I don't know what that is.
"and other fields sent on the scoring signals request" is a little confusing (I thought it was attached to the first half of the sentence).  Maybe it might be clearer to write "...which isn't captured in other fields sent on the scoring signals fetch request (e.g. `renderURL`)."